src/components/MoneyRequestConfirmationList.js

```diff
@@ -499,6 +520,16 @@ function MoneyRequestConfirmationList(props) {
                             disabled={didConfirm || props.isReadOnly}
                         />
                     )}
+                    {canUseTags && !!tagList && (
```

Comment:
I just noticed this line is incorrect, we should use `!_.isEmpty(tagList)` instead of `!!tagList`. @BeeMargarida can you add that change to your next PR?

Reply:
Yap!